examples/graphene.jl

```diff
@@ -1,30 +1,55 @@
-#
-# Example of a slightly larger graphene calculation.
-#src tags: long
-#
+# # Graphene band structure
+
+# This example plots the band structure of graphene, a 2D material. 2D band structures are not supported natively (yet), so we manually build a custom path in reciprocal space.
 
 using DFTK
-setup_threading()
+using Unitful
+using UnitfulAtomic
 
-kgrid = [12, 12, 4]
+# Define the convergence parameters (these should be increased in production)
```

Comment:
Keep in mind that unless you use `##` literate will start a new evaluation box with the last result printed to the user. I think at places in this file you don't want that.